### PR TITLE
Fixing issue #322

### PR DESCRIPTION
**Pull request by shridharkalagi**

### CODE REVIEW

**Comment by SrinivasanTarget** [comment]
I don't see any check related to app extensions...is it missed or pr is not yet ready for review?

File: src/main/java/com/appium/utils/CapabilityManager.java [hunk 1]
```diff
@@ -80,6 +80,11 @@ public boolean isSimulatorAppPresentInCapsJson() {
         return hasApp && getCapabilityObjectFromKey("iOS").getJSONObject("app").has("simulator");
     }
 
+    public boolean isRealDeviceAppPresentInCapsJson() {
+        boolean hasApp = getCapabilityObjectFromKey("iOS").has("app");
+        return hasApp && getCapabilityObjectFromKey("iOS").getJSONObject("app").has("device");
```
Comment (saikrishna321) [inline src/main/java/com/appium/utils/CapabilityManager.java]: Must have: return device prop if true

[the remark this answers is not in the document]
Comment (SrinivasanTarget) [reply src/main/java/com/appium/utils/CapabilityManager.java]: looks fine

Comment (SrinivasanTarget) [reply src/main/java/com/appium/utils/CapabilityManager.java]: device prop?

**Pull request merged**